packages/pg-v5/commands/copy.js

```diff
@@ -31,9 +40,10 @@ function * run (context, heroku) {
         heroku.get(`/apps/${attachment.app.name}/config-vars`)
       ]
       attachment.addon = addon
+      config = upperCaseConfig(config) // Upper case config var keys
       return {
         name: attachment.name.replace(/^HEROKU_POSTGRESQL_/, '').replace(/_URL$/, ''),
-        url: config[attachment.name + '_URL'],
+        url: config[attachment.name.toUpperCase() + '_URL'], // Upper case attachment name
```

Comment:
This could also be `findURL(config, attachment.name)` which avoids mutations...

Reply:
Hmm, that is a better call while avoiding the mutation. I'll see what this looks like.

Reply:
`upperCaseConfig` returns a new object anyway, so NBD as Spiderman would say.